src/main/java/com/harium/suneidesis/chat/box/BaseChatBox.java

```diff
@@ -20,9 +23,15 @@ public abstract class BaseChatBox implements ChatBox, Parser {
     protected List<Parser> parsers = new ArrayList<>();
     protected List<Interceptor> interceptors = new ArrayList<>();
 
+    public BaseChatBox() {
```

Comment:
I prefer leaving the BaseChatBox as empty as possible, not every person need all the interceptors and they add overhead during execution.